src/parser/mod.rs

```diff
@@ -204,13 +202,13 @@ pub struct FragmentHeader<'a> {
     ///
     /// All fragments without a name will have a `name_reference` of 0.
     /// The one exception being the 0x35 fragment which will always reference 0xFF000000.
-    pub name_reference: i32,
+    pub name_reference: Option<StringReference>,
```

Comment:
We have an actual `StringReference` type now. And at that point it made more sense to make this optional.